rackcity/models/it_instance.py

```diff
@@ -1,10 +1,23 @@
+import re
+from django.core.exceptions import ValidationError
 from django.db import models
 from .it_model import ITModel
 from .rack import Rack
 
 
+def validate_hostname(value):
+    hostname_pattern = re.compile("[a-z]([a-z0-9-]{0,61}[a-z0-9])?")
```

Comment:
added `( .. )?` around the last group so it needs 0 or more of the last shindig

Reply:
0 or *1 but the ? is correct

Reply:
Buuuut I think capital letters are allowed sorry I didn't flag the first time